content/en/monitors/guide/composite_use_cases.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Composite Monitor Use Cases
+further_reading:
+- link: "monitors/types/composite/"
+  tag: "Documentation"
+  text: "Composite monitor type"
+---
+
+
+## Overview 
+
+This guide lists non-exhaustive use cases for composite monitors. These examples illustrate how composite monitors can be configured to address various use cases in monitoring environments:
+
+- [Error rates](#error-rates)
+- [Monitor frequent metrics](#monitor-frequent-metrics)
+- [Step monitor](#step-monitor)
+- [Renotifying on recovery](#renotifying-on-recovery)
+- [Delay on notification](#delay-on-notification)
+
+
+## Error rates
+
+Alert when the error rate exceeds a threshold only when hits are above a certain number.
+
+Create two monitors:
+
+- **Monitor A**: Alert when `trace.requests.request.errors / trace.requests.request.hits > X`
+- **Monitor B**: Alert when `trace.requests.request.hits > Y`
+
+**Composite Monitor C**: Alert when both Monitor A and Monitor B are in alert (A && B).
```

Comment:
```suggestion
**Composite Monitor C**: Alert when both Monitor A and Monitor B are in alert (A & B).
```
is there a duplicate ampersand here ? Edit I see this used in later examples so I see it is a condition, do disregard :)